dist/uPlot.d.ts

```diff
@@ -200,6 +200,11 @@ export type DateFormatterFactory = (tpl: string) => (date: Date) => string;
 
 export type LocalDateFromUnix = (ts: number) => Date;
 
+export enum DrawOrderOption {
```

Comment:
`const enum` is preferable: https://www.typescriptlang.org/docs/handbook/enums.html#const-enums

Reply:
actually, you're right. it needs some suffix. leave it as-is and i'll think about what i like better that "Option", if anything.